_posts/2016-09-01-akka-community-survey-2016.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "Akka Community Survey 2016"
+description: ""
+author: Markus Eisele
+category: community
+tags: [community,contributing,akka,survey]
+---
+{% include JB/setup %}
+
+Dear hakkers,
+
+we did a lot this year already. Everything started with revamping this team blog. We want to share more technical details with you and introduce you to the latest happenings. We also started to recognize our most active contributors by giving out special t-shirts. But this shouldn't be the last thing we do this year. Today, we're announcing the "Akka Community Survey 2017".
+
+To further foster the adoption of Akka we want to make sure to give you, our community, a voice. We compiled a list of questions which will help us to identify missing bits and pieces but also understand better how you are using Akka.
+
+It asks about module ussage, your favorite language for Akka, areas of improvements and will take you probably just short 10 minutes. We keep it open for a week (until September 8th, 2016) and hope for plenty of responses. So make sure to:
+
+*[Take the survey today!](https://lightbend.qualtrics.com/SE/?SID=SV_3VnWxXR09f7lAEd)*
+
+As a little motivation, we're raffling away 100 Akka t-shirts. Every response who enters the raffle has a chance to win one. The only prerequisite is, that you give us your email address at the end of the survey. But you decide: You can take this anonymous survey without entering the drawing.
+Winners will be notified within two weeks after we closed the survey.
+
+![Win an Akka T-Shirt]({{ site.url }}/assets/akka-shirt.png)
+
+Now, [take the survey today](https://lightbend.qualtrics.com/SE/?SID=SV_3VnWxXR09f7lAEd), spread the word on your social media channels and help get plenty of responses for your Akka Team!
```

Comment:
This sounds too much "Like us on Facebook". I would like to tone this down a bit.